hieradata/cluster/luan.yaml

```diff
@@ -1,4 +1,46 @@
 ---
 clustershell::groupmembers:
   luan: {group: "luan", member: "luan[01-05]"}
+profile::core::ospl::enable_rundir: true
+profile::core::k8snode::enable_dhcp: true
 tuned::active_profile: "latency-performance"
+
+nm::connections:
+  eno1np0:
+    content:
+      connection:
+        id: "eno1np0"
+        uuid: "8d5f0c26-a6c6-3704-92a5-c347e9fbbf28"
+        type: "ethernet"
+        interface-name: "eno1np0"
+      ethernet: {}
+      ipv4:
+        method: "auto"
```

Comment:
We should probably be using bridge interfaces for all baremetal clusters for consistency but as this is a rush, it is OK to use the physical interface this time.  We should plan to rebuild this cluster again and convert to using LAGs.